src/embedded.ts

```diff
@@ -108,11 +108,21 @@ export class RoomWidgetClient extends MatrixClient {
     private lifecycle?: AbortController;
     private syncState: SyncState | null = null;
 
+    /**
+     *
+     * @param widgetApi - the widget api to use for the embedded "fake" client
+     * @param opts - Client related options
+     * @param sendContentLoaded - Whether to send a content loaded event.
+     *   Set to false if: the widget uses waitForIFrameLoad=true, or if the
+     *   the widget wants to send the ContentLoaded action at a later point in time
+     *   after initial setup.
```

Comment:
Is there a difference between a "content loaded event" and a "ContentLoaded action"? If not, could you use the same name consistently? I couldn't really find much documentation about the widget API so I don't really know what they mean. Is there any documentation that we could link to? Or can we at least use the full name of a symbol in the matrix-widget-api?

Please proofread this yourself: there are two "the"s in the sentence.

Suggest more use of backticks around things that are names of javascript identifiers, such as `false` and `waitForIFrameLoad=true`

Reply:
thanks. We should also always call it an action to make it clear if we refer to a widget message (=action) or a matrix message/event (=event).